Justfile

```diff
@@ -17,4 +17,21 @@ test-go:
     go test ./... -v
 
 start:
-    go run ./...
\ No newline at end of file
+    go run ./...
+
+clean-lib:
+    rm -rf lib
+
+checkout-optimism-monorepo:
+    rm -rf lib/optimism
+    mkdir -p lib/optimism && \
+    cd lib/optimism && \
+    git init && \
+    git remote add origin https://github.com/ethereum-optimism/optimism.git && \
+    git fetch --depth=1 origin 9001eef4784dc2950d0bdcda29752cb2939bae2b && \
+    git reset --hard FETCH_HEAD && \
+    git submodule update --init --recursive && \
+    make cannon-prestate
+
+generate-genesis: checkout-optimism-monorepo
+    python3 scripts/generate-genesis.py
```

Comment:
It doesn't need to be in this PR more of something for us to think about. This will require everyone to also have `make` and `python3` setup already which we all do but should consider in the long run if we can get away with pulling in the monorepo in a docker container generating the files in the container and outputting them to our host fs. Would also love to get this out of python at some point but get why we're doing it this why since the monorepo does it this way.

Reply:
Yeah I agree. We should reduce dependencies that folks need to download wherever possible. Could be good to make a follow-on issue for this.

Reply:
Yep! agreed that there are way better approaches. Docker approach makes sense to me.

I don't think we should do much in the near term because
- the config/genesis generation process is currently in flux
- we're going to consolidate the process with some of the code in superchain registry
- this will be run once every few months by a single dev, then committed.